bokeh/core/_templates/autoload_js.js

```diff
@@ -23,15 +23,17 @@ calls it with the rendered model.
 :param css_urls: CSS urls to inject
 :type css_urls: list
 
-
 #}
 (function(global) {
   function now() {
     return new Date();
   }
 
-  if (typeof (window._bokeh_onload_callbacks) === "undefined") {
+  var force = "{{ force }}";
+
+  if (typeof (window._bokeh_onload_callbacks) === "undefined" || force !== "") {
```

Comment:
This empty string business seems pretty lame, but AFAIK there's no way to have a "default" value for a template param?